src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -62,39 +61,6 @@ public void testSimpleConnection() throws SQLException {
     con.close(); // ensure no exception
   }
 
-  @Test
```

Comment:
Please ignore- removing unneccessary test that got merged by accident and is causing failures.